### ISSUE TITLE
navbar

### ISSUE THREAD

**Issue by darshil-geek** [PES-Innovation-Lab/web#67]
the navbar isn't completely sticky, on loading the page if we scroll down and then up again, then the navbar disappears

**Comment by souravtecken** [comment]
Hi @darshil-geek. The idea was that the navbar slides away from view when the user's scrolling down and pops back in when the user scrolls up. This was added in with #53. I hope the behavior is consistent with what I've mentioned, or were you referring to something else?

**Comment by souravtecken** [comment]
Now that I notice it, it would be cleaner if the navbar did toggle back in when the user stops scrolling too, rather than having to scroll back up. Would you want to take up this issue @darshil-geek?

**Comment by darshil-geek** [comment]
Yes sure, but my end sem exams are going on now, so I'll do it later
Trust that's fine ? @souravtecken

**Comment by souravtecken** [comment]
Yeah sure 👍
